pom.xml

```diff
@@ -109,21 +110,20 @@
             <artifactId>springfox-swagger-ui</artifactId>
             <version>2.6.1</version>
         </dependency>
-        <!-- <dependency>
+        <dependency>
             <groupId>javax.el</groupId>
             <artifactId>javax.el-api</artifactId>
-            <version>3.0.0</version>
-        </dependency> -->
+            <version>2.2.4</version>
+        </dependency>
         <dependency>
-            <groupId>org.glassfish</groupId>
+            <groupId>org.glassfish.web</groupId>
             <artifactId>javax.el</artifactId>
-            <!-- version>2.2.4</version> -->
-            <version>3.0.1-b08</version>
+            <version>2.2.4</version>
         </dependency>
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.4</version>
+            <scope>provided</scope>
```

Comment:
Lombok version seems needed (in my case it won't build) but also it allows dependabot to provide suggestions.

Reply:
OK; I'll fix that one too.